.github/workflows/ci.yml

```diff
@@ -81,11 +81,7 @@ jobs:
           - mpi
           - threaded
         include:
-          - version: '1.8'
-            os: ubuntu-latest
-            arch: x64
-            trixi_test: threaded_legacy
-          - version: '1.9'
+          - version: '1.11'
             os: ubuntu-latest
             arch: x64
             trixi_test: threaded_legacy
```

Comment:
Why did you decide to use 1.11 for this?

Reply:
Since otherwise `threaded_legacy` remains unused. I thought this would be an initial and small step towards testing at least some parts on v1.11. But I can also just remove it, I have no strong opinion here either way